pkg/policy/opa/engine_test.go

```diff
@@ -1 +1,98 @@
 package opa
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFilterRules(t *testing.T) {
+	testPolicyPath := "test"
+
+	type args struct {
+		e          *Engine
+		policyPath string
+		scanRules  []string
+		skipRules  []string
+	}
+	tests := []struct {
+		name        string
+		args        args
+		assert      bool
+		regoMapSize int
+	}{
+		{
+			name: "no scan and skip rules",
+			args: args{
+				e: &Engine{},
+			},
+		},
+		{
+			name: "scan rules test",
+			args: args{
+				e: &Engine{
+					regoDataMap: getTestRegoDataMap(10),
+				},
+				policyPath: testPolicyPath,
+				scanRules:  []string{"Rule.0", "Rule.1", "Rule.2", "Rule.3", "Rule.11"},
+			},
+			assert:      true,
+			regoMapSize: 4,
+		},
+		{
+			name: "scan rules not found in path",
+			args: args{
+				e: &Engine{
+					regoDataMap: getTestRegoDataMap(10),
+				},
+				policyPath: testPolicyPath,
+				scanRules:  []string{"Rule.11", "Rule.12", "Rule.13"},
+			},
+			assert:      true,
+			regoMapSize: 0,
+		},
+		{
+			name: "skip rules test",
```

Comment:
Can we have a test for `scan and skip rules present`?

Reply:
added a test for it.